client/state/site-settings/exporter/actions.js

```diff
@@ -83,28 +96,32 @@ export function advancedSettingsFail( siteId, error ) {
  * @param  {Number}   siteId  The ID of the site to export
  * @return {Function}         Action thunk
  */
-export function startExport( siteId ) {
-	return ( dispatch ) => {
+export function startExport( siteId, exportAll = true ) {
```

Comment:
Although it's just one argument and defaults to a sane value, could we use something other than a boolean value to indicate the change of behavior? How about an options array?

```js
export function startExport( siteId, { exportAll } ) {
	…
}

exportAll( siteId, { exportAll: true } );
```